src/wombats/scheduler/core.clj

```diff
@@ -44,3 +46,44 @@
 ;; TODO #349
 ;; Schedule active & active intermission games
 ;; in the case of system restart.
+
+(defn add-game-scheduler
+  "Called by the scheduler to add a game that starts a day after the scheduled time"
+  [{:keys [initial-time
+           game-params
+           add-game-fn
+           gen-id-fn
+           get-game-by-id-fn
+           get-arena-by-id-fn
+           start-game-fn]}]
```

Comment:
A way to further clean this up would be:

```
[{:keys [start-game-fn]
   {:keys [arena-id game]} :game-params}]
```

Then you could get rid of `game-params` and the accessors down below.